helm/nifi-cluster/values.yaml

```diff
@@ -51,6 +51,10 @@ cluster:
     labels: {}
     # -- host aliases to assign to each pod. See: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.23/#hostalias-v1-core
     hostAliases: []
+    # -- (object) The pod readiness probe override: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-readiness-probes
+    readinessProbe: 
```

Comment:
Does leaving these as `nil` fall back to the default provided by the operator?
If so, this LGTM!

Reply:
Yes. [Here for readiness](https://github.com/konpyutaika/nifikop/blob/master/pkg/resources/nifi/pod.go#L429-L432) and [here for liveness](https://github.com/konpyutaika/nifikop/blob/master/pkg/resources/nifi/pod.go#L445-L448)